src/ucp/wireup/address.c

```diff
@@ -132,50 +135,30 @@ static uint64_t ucp_worker_iface_can_connect(uct_iface_attr_t *attrs)
 }
 
 /* Pack a string and return a pointer to storage right after the string */
-static void* ucp_address_pack_worker_name(ucp_worker_h worker, void *dest,
-                                          uint64_t flags)
+static void* ucp_address_pack_worker_name(ucp_worker_h worker, void *dest)
 {
-#if ENABLE_DEBUG_DATA
     const char *s;
     size_t length;
 
-    if (!(flags & UCP_ADDRESS_PACK_FLAG_WORKER_NAME)) {
-        return dest;
-    }
-
     s      = ucp_worker_get_name(worker);
     length = strlen(s);
     ucs_assert(length <= UINT8_MAX);
     *(uint8_t*)dest = length;
     memcpy(UCS_PTR_TYPE_OFFSET(dest, uint8_t), s, length);
     return UCS_PTR_BYTE_OFFSET(UCS_PTR_TYPE_OFFSET(dest, uint8_t), length);
-#else
-    return dest;
-#endif
 }
 
 /* Unpack a string and return pointer to next storage byte */
-static const void* ucp_address_unpack_worker_name(const void *src, char *s,
-                                                  size_t max, uint64_t flags)
+static const void*
+ucp_address_unpack_worker_name(const void *src, char *s)
 {
-#if ENABLE_DEBUG_DATA
     size_t length, avail;
 
-    if (!(flags & UCP_ADDRESS_PACK_FLAG_WORKER_NAME)) {
-        s[0] = '\0';
-        return src;
-    }
-
-    ucs_assert(max >= 1);
     length   = *(const uint8_t*)src;
-    avail    = ucs_min(length, max - 1);
+    avail    = ucs_min(length, UCP_WORKER_NAME_MAX - 1);
```

Comment:
we have config var in the context for max worker name

Reply:
but this is the remote worker name, no need to limit (in case different remote processes have different configs for worker name size)